share/module_model_constants.F

```diff
@@ -17,81 +17,81 @@ MODULE module_model_constants
 
    REAL    , PARAMETER :: asselin      = .025
 !   REAL    , PARAMETER :: asselin      = .0
    REAL    , PARAMETER :: cb           = 25.
 
-   REAL    , PARAMETER :: XLV0         = 3.15E6
-   REAL    , PARAMETER :: XLV1         = 2370.
-   REAL    , PARAMETER :: XLS0         = 2.905E6
-   REAL    , PARAMETER :: XLS1         = 259.532
+   REAL    , PARAMETER :: XLV0         = 3.15E6       !  constant defined for calculation of latent heating 
+   REAL    , PARAMETER :: XLV1         = 2370.        !  constant defined for calculation of latent heating 
+   REAL    , PARAMETER :: XLS0         = 2.905E6      !  constant defined for calculation of latent heating
+   REAL    , PARAMETER :: XLS1         = 259.532      !  constant defined for calculation of latent heating
 
-   REAL    , PARAMETER :: XLS          = 2.85E6
-   REAL    , PARAMETER :: XLV          = 2.5E6
-   REAL    , PARAMETER :: XLF          = 3.50E5
+   REAL    , PARAMETER :: XLS          = 2.85E6      ! latent heat of sublimation of water at 0^oC (J kg^-1) 
+   REAL    , PARAMETER :: XLV          = 2.5E6       ! latent heat of vaporization of water at 0^oC (J kg^-1)
+   REAL    , PARAMETER :: XLF          = 3.50E5      ! latent heat of fusion of water at 0^oC (J kg^-1)
 
-   REAL    , PARAMETER :: rhowater     = 1000.
-   REAL    , PARAMETER :: rhosnow      = 100.
-   REAL    , PARAMETER :: rhoair0      = 1.28
+   REAL    , PARAMETER :: rhowater     = 1000.       ! density of liquid water at 0^oC (kg m^-3)
+   REAL    , PARAMETER :: rhosnow      = 100.        ! density of snow (kg m^-3)
+   REAL    , PARAMETER :: rhoair0      = 1.28        ! density of dry air at 0^oC and 1000mb pressure (kg m^-3)
 !
 ! Now namelist-specified parameter: ccn_conc - RAS
 !   REAL    , PARAMETER :: n_ccn0       = 1.0E8
 !
-   REAL    , PARAMETER :: piconst      = 3.1415926535897932384626433
-   REAL    , PARAMETER :: DEGRAD       = piconst/180.
+   REAL    , PARAMETER :: piconst      = 3.1415926535897932384626433    ! constant of PI
+   REAL    , PARAMETER :: DEGRAD       = piconst/180.                   ! radians per degree
    REAL    , PARAMETER :: DPD          = 360./365.
 
-   REAL    , PARAMETER ::  SVP1=0.6112
-   REAL    , PARAMETER ::  SVP2=17.67
-   REAL    , PARAMETER ::  SVP3=29.65
-   REAL    , PARAMETER ::  SVPT0=273.15
-   REAL    , PARAMETER ::  EP_1=R_v/R_d-1.
-   REAL    , PARAMETER ::  EP_2=R_d/R_v
-   REAL    , PARAMETER ::  KARMAN=0.4
-   REAL    , PARAMETER ::  EOMEG=7.2921E-5
-   REAL    , PARAMETER ::  STBOLT=5.67051E-8
-
-   REAL    , PARAMETER ::  prandtl = 1./3.0
-                                         ! constants for w-damping option
-   REAL    , PARAMETER ::  w_alpha = 0.3 ! strength m/s/s
-   REAL    , PARAMETER ::  w_beta  = 1.0 ! activation cfl number
-
-       REAL , PARAMETER ::  pq0=379.90516
-       REAL , PARAMETER ::  epsq2=0.2
+   REAL    , PARAMETER ::  SVP1=0.6112      ! constant for saturation vapor pressure (dimensionless)
```

Comment:
For the following three, it is probably better to say 'constant for saturation vapor pressure calculation (dimensionless)'.